### PR TITLE
dev, v3.0: add tidb-in-kubernetes file in the monitoring section

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

add tidb-in-kubernetes file in the monitoring section

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1552

### Which version does your change affect? 

dev, v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/how-to/monitor/tidb-in-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl port-forward -n ${namespace} svc/${releaseName}-grafana 3000:3000 &>/tmp/portforward-grafana.log
+```
+
+Then open [http://localhost:3000](http://localhost:3000) in your browser and the default username and password are `admin`.
+
+The Grafana service is exposed by `NodePort` by default. If the Kubernetes cluster supports the load balancer, you can change `monitor.grafana.service.type` to `LoadBalancer` in `values.yaml` and then access the dashboard through the load balancer after executing `helm upgrade`.
```
Comment (AstroProfundis) [inline dev/how-to/monitor/tidb-in-kubernetes.md]:
```suggestion
The Grafana service is exposed by `NodePort` by default. If the Kubernetes cluster supports load balancer, you can change `monitor.grafana.service.type` to `LoadBalancer` in `values.yaml` and then access the dashboard through the load balancer after executing `helm upgrade`.
```

File: dev/how-to/monitor/tidb-in-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl port-forward -n ${namespace} svc/${releaseName}-prometheus 9090:9090 &>/tmp/portforward-prometheus.log
+```
+
+Then open [http://localhost:9090](http://localhost:9090) in your browser or access this address via a client tool.
+
+The Prometheus service is exposed by `NodePort` by default. If the Kubernetes cluster supports the load balancer, you can change `monitor.prometheus.service.type` to `LoadBalancer` in `values.yaml` and then access the monitoring data through the load balancer after executing `helm upgrade`.
```
Comment (AstroProfundis) [inline dev/how-to/monitor/tidb-in-kubernetes.md]:
```suggestion
The Prometheus service is exposed by `NodePort` by default. If the Kubernetes cluster supports load balancer, you can change `monitor.prometheus.service.type` to `LoadBalancer` in `values.yaml` and then access the monitoring data through the load balancer after executing `helm upgrade`.
```

File: dev/how-to/monitor/tidb-in-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl port-forward -n ${namespace} svc/${releaseName}-prometheus 9090:9090 &>/tmp/portforward-prometheus.log
+```
+
+Then open [http://localhost:9090](http://localhost:9090) in your browser or access this address via a client tool.
+
+The Prometheus service is exposed by `NodePort` by default. If the Kubernetes cluster supports load balancer, you can change `monitor.prometheus.service.type` to `LoadBalancer` in `values.yaml` and then access the monitoring data through the load balancer after executing `helm upgrade`.
```
Comment (dcalvin) [inline dev/how-to/monitor/tidb-in-kubernetes.md]:
```suggestion
The Prometheus service is exposed by `NodePort` by default. If the Kubernetes cluster supports load balancer, you can change `monitor.prometheus.service.type` to `LoadBalancer` in `values.yaml`. Then, after executing `helm upgrade`, access the monitoring data through the load balancer.
```

File: dev/how-to/monitor/tidb-in-kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+### Monitor the host
+
+Monitoring the host and its resources works in the same way as monitoring physical resources of a traditional server.
+
+If you already have a monitoring system for your physical server in your existing infrastructure, you only need to add the host that holds Kubernetes to the existing monitoring system by conventional means; if there is no monitoring system available, or you want to deploy a separate monitoring system to monitor the host that holds Kubernetes, then you can use any monitoring system that you are familiar with.
+
+The newly deployed monitoring system can run on a separate server, run directly on the host that holds Kubernetes, or run in a Kubernetes cluster. With different deployment methods, there might be some differences in the deployment configuration and resource utilization, but there is no major difference in usage.
```
Comment (dcalvin) [inline dev/how-to/monitor/tidb-in-kubernetes.md]:
```suggestion
The newly deployed monitoring system can run on a separate server, directly on the host that holds Kubernetes, or in a Kubernetes cluster. Different deployment methods might mean differences in the deployment configuration and resource utilization, but there are no major differences in usage.
```

File: dev/how-to/monitor/tidb-in-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+Common open source monitoring systems that can be used to monitor server resources are:
```
Comment (dcalvin) [inline dev/how-to/monitor/tidb-in-kubernetes.md]:
```suggestion
Some common open source monitoring systems that can be used to monitor server resources are:
```

File: dev/how-to/monitor/tidb-in-kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+- [CollectD](https://collectd.org/)
+- [Nagios](https://www.nagios.org/)
+- [Prometheus](http://prometheus.io/) & [node_exporter](https://github.com/prometheus/node_exporter)
+- [Zabbix](https://www.zabbix.com/)
+
+Some cloud service providers or specialized performance monitoring service providers also have their own free or chargeable monitoring solutions to choose from.
```
Comment (dcalvin) [inline dev/how-to/monitor/tidb-in-kubernetes.md]:
```suggestion
Some cloud service providers or specialized performance monitoring service providers also have their own free or chargeable monitoring solutions that you can choose from.
```

File: dev/how-to/monitor/tidb-in-kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+It is recommended to deploy a host monitoring system in the Kubernetes cluster via [Prometheus Operator](https://github.com/coreos/prometheus-operator) based on [Node Exporter](https://github.com/prometheus/node_exporter) and Prometheus. This solution can also be compatible with and used for monitoring the Kubernetes' own components.
```
Comment (dcalvin) [inline dev/how-to/monitor/tidb-in-kubernetes.md]:
```suggestion
It is recommended to deploy a host monitoring system in the Kubernetes cluster via [Prometheus Operator](https://github.com/coreos/prometheus-operator) based on [Node Exporter](https://github.com/prometheus/node_exporter) and Prometheus. This solution can also be compatible with and used for monitoring the Kubernetes' own components.
```

File: dev/how-to/monitor/tidb-in-kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+### Monitor Kubernetes components
+
+For monitoring Kubernetes components, you can refer to the solutions provided in the [Kubernetes official documentation](https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/) or use other Kubernetes-compatible monitoring systems.
+
+Some cloud service providers may provide their own solutions for monitoring Kubernetes components, and some specialized performance monitoring service providers have their own Kubernetes integration solutions to choose from.
```
Comment (dcalvin) [inline dev/how-to/monitor/tidb-in-kubernetes.md]:
```suggestion
Some cloud service providers may provide their own solutions for monitoring Kubernetes components, and some specialized performance monitoring service providers have their own Kubernetes integration solutions that you can choose from.
```

**Comment by yikeke** [comment]
Comments addressed. PTAL again. @dcalvin

**Pull request merged**